sensor_array/analysis/execute_brute_force_analysis.py

```diff
@@ -6,7 +6,7 @@
 import sys
 import pandas as pd
 from datetime import datetime
-from mof_array.pmf.process_mass_data import (
+from brute_force_analysis import (
```

Comment:
Changes need to be atomic. You can't change the module name without renaming the module.